dhis-2/dhis-api/src/main/java/org/hisp/dhis/program/notification/NotificationPagingParam.java

```diff
@@ -46,5 +46,8 @@ public class BaseNotificationParam {
 
   private Integer pageSize;
 
+  @Deprecated(since = "2.41")
```

Comment:
This is not really a request param, right?
So it doesn't make sense to deprecate it

Reply:
https://github.com/dhis2/dhis2-core/blob/b59d61119be81e63b681d64e474155bb242e7afc/dhis-2/dhis-web-api/src/main/java/org/hisp/dhis/webapi/controller/event/ProgramNotificationTemplateController.java#L86

Reply:
As I was saying, we should deprecate only fields that are actually exposed to the API.
This is an internal object created by some mapping, isn't it?